spec/tasks/metrics_spec.rb

```diff
@@ -9,7 +9,7 @@
   describe "#overview" do
     let(:event_name) { "Admin Overview Link Clicked" }
     let(:click_target) { "https://forem.gitbook.io/forem-admin-guide/quick-start-guide" }
-    let(:host) { ENV['APP_DOMAIN'] }
+    let(:host) { ENV["APP_DOMAIN"] }
```

Comment:
This is almost certainly me, sorry. There's something not firing with rubocop in my commit hooks.